docs-website/sidebars.js

```diff
@@ -184,12 +192,6 @@ module.exports = {
       // TODO: the titles of these should not be in question form in the sidebar
       "docs/developers",
       "docs/docker/development",
-      "metadata-ingestion/adding-source",
-      {
-        type: "doc",
-        label: "Ingesting files from S3",
-        id: "metadata-ingestion/source_docs/s3",
```

Comment:
I think we need this one

Reply:
S3 doc is present under Sources. Why do we need this specifically here?